sycl/doc/extensions/README.md

```diff
@@ -1,3 +1,47 @@
+| [Queue Order Properties](OrderedQueue/OrderedQueue_v2.adoc)                                                                 | Supported                               | |
+| [Queue Shortcuts](QueueShortcuts/QueueShortcuts.adoc)                                                                       | Supported                               | |
+| [Reductions for ND-Range Parallelism](Reduction/Reduction.md)                                                               | Partially supported(OpenCL: CPU, GPU)   | Not supported: multiple reduction vars, USM reduction vars, multi-dimensional reduction vars |
+| [SYCL_INTEL_relax_standard_layout](RelaxStdLayout/SYCL_INTEL_relax_standard_layout.asciidoc)                                | Supported                               | |
+| [SYCL_INTEL_reqd_work_group_size](ReqdWorkGroupSize/SYCL_INTEL_reqd_work_group_size.asciidoc)                               | Supported(OpenCL: CPU, GPU)             | |
+| [SPV_INTEL_function_pointers](SPIRV/SPV_INTEL_function_pointers.asciidoc)                                                   | Supported(OpenCL: CPU, GPU; HOST)       | |
+| [SPV_INTEL_inline_assembly](SPIRV/SPV_INTEL_inline_assembly.asciidoc)                                                       | Supported(OpenCL: GPU)                  | |
+| [SYCL_INTEL_static_local_memory_query](StaticLocalMemoryQuery/SYCL_INTEL_static_local_memory_query.asciidoc)                | Proposal                                | |
+| [SYCL_INTEL_sub_group_algorithms](SubGroupAlgorithms/SYCL_INTEL_sub_group_algorithms.asciidoc)                              | Partially supported(OpenCL: CPU, GPU)   | Features from SYCL_INTEL_group_algorithms extended to sub-groups |
+| [Sub-groups for NDRange Parallelism](SubGroupNDRange/SubGroupNDRange.md)                                                    | Deprecated(OpenCL: CPU, GPU)            | |
+| [SYCL_INTEL_unnamed_kernel_lambda](UnnamedKernelLambda/SYCL_INTEL_unnamed_kernel_lambda.asciidoc)                           | Supported(OpenCL)                       | |
+| [Unified Shared Memory](USM/USM.adoc)                                                                                       | Supported(OpenCL)                       | |
+
+Legend:
+
+|  Keyword    |   Meaning   |
+|-------------|:------------|
+|  Proposal                        | A document describing an extension is published, but the extension is not supported yet |
+|  Supported                       | An extension is supported |
+|  Partially supported             | An extension is partially supported, see comments column for more info |
+|  Deprecated                      | An extension is deprecated and can be removed in future versions |
+|  (API: DeviceType1, DeviceType2) | An extension is supported when specific combination of API and device types are used. If device type or API are not mentioned then an extension is supported on any device type or API |
```

Comment:
I don't feel strongly about this, but would it make sense to point out somewhere what the valid values for "API" and "DeviceType" are?  Or call out host support separately?

It took me a while to realize that the difference between "OpenCL: CPU, GPU, FPGA" and "OpenCL" is that the latter supports the host device.  It wasn't obvious to me which device type was missing until I compared against the row for function pointers.

Reply:
Yeah, you are right. Actually OpenCL should not include Host device. Added valid values and made host to be an API. It sounds strange, but do know how to make it better.

Reply:
This is clearer, thanks.  I agree that Host as an API is the best option we have right now.